<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="4" tests="11" name="USER_Admin_Reports.User_SMR_Bot_Status_reports" time="60.612" errors="0" timestamp="2024-12-11T18:24:34 IST" skipped="0">
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Username" time="0.881"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Password" time="1.025"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Captcha" time="0.714"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="SUBMIT_Button" time="0.162"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Remove_PouP" time="0.672"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Click_On_Reports" time="3.762"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Click_On_SMR_Bot_Status" time="0.093"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Check_Data_Workshop_Wise" time="18.308">
    <failure message="&#039;Workshop Wise&#039; Admin Reports does not contain any data it is  null." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 'Workshop Wise' Admin Reports does not contain any data it is  null.
at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.checkDataPresence(User_SMR_Bot_Status_reports.java:332)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.Check_Data_Workshop_Wise(User_SMR_Bot_Status_reports.java:74)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Check_Data_Workshop_Wise -->
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Check_Data_Day_Wise" time="17.550">
    <failure message="&#039;Data Day Wise&#039; Admin Reports does not contain any data it is  null." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 'Data Day Wise' Admin Reports does not contain any data it is  null.
at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.checkDataPresence(User_SMR_Bot_Status_reports.java:332)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.Check_Data_Day_Wise(User_SMR_Bot_Status_reports.java:145)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Check_Data_Day_Wise -->
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Check_Service_Type_Wise" time="17.428">
    <failure message="&#039;Service Type Wise&#039; Admin Reports does not contain any data it is  null." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 'Service Type Wise' Admin Reports does not contain any data it is  null.
at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.checkDataPresence(User_SMR_Bot_Status_reports.java:332)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.Check_Service_Type_Wise(User_SMR_Bot_Status_reports.java:214)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Check_Service_Type_Wise -->
  <system-out/>
  <testcase classname="USER_Admin_Reports.User_SMR_Bot_Status_reports" name="Comparison_Of_All_Tab_Text" time="0.017">
    <failure message="Test failed with the following issues:
Workshop Wise Texts is empty or null.
Day Wise Texts is empty or null.
Service Type Texts is empty or null.
" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test failed with the following issues:
Workshop Wise Texts is empty or null.
Day Wise Texts is empty or null.
Service Type Texts is empty or null.

at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.User_SMR_Bot_Status_reports.Comparison_Of_All_Tab_Text(User_SMR_Bot_Status_reports.java:308)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Comparison_Of_All_Tab_Text -->
  <system-out/>
</testsuite> <!-- USER_Admin_Reports.User_SMR_Bot_Status_reports -->
